content/SCALECLIReference/Auth/CLIApiKey.md

```diff
@@ -10,6 +10,193 @@ tags:
 
 {{< toc >}}
 
+## API_Key Namespace
+
+The *api_key* namespace has five commands and is based on API key management functions found in the SCALE API and web UI.
+It provides access to API kay creation and management methods through the **api_key** commands.
+
+## API_Key Commands
+
+The following **api_key** commands allow you to create, view, and delete API keys.
+
+You can enter commands from the main CLI prompt or from the snapshot namespace prompt.
+
 {{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
 
-{{< taglist tag="scalecliauth" limit="10" title="Related CLI Auth Articles" >}}
\ No newline at end of file
+### Interactive Argument Editor (TUI)
+
+{{< include file="/_includes/CLI/HintInteractiveArgsEditor.md" type="page" >}}
+
+### Create Command
+
+The `create` command lets you create simple or complex API keys.
+
+{{< expand "Creating a Simple API Key" "v" >}}
+
+#### Description
+The `create` command requires the `name` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns the API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key create name=<i>name</i></code>
+
+Where *name* is the name you want to assign to the key.
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key create name=apikey3
+API Key: 3-xTqwhyf3SrUgUlotMQEEGuUr6oRvqg89SBDfXob6xtWSgLbRiDBr6SVRWxswSXx3
+```
+{{< /expand >}}
+{{< /expand >}}
+
+{{< expand "Creating a Complex API Key" "v" >}}
+
+#### Description
+The `create` command requires the `name` property. You can also specify an HTTP method and a WebSocket API method.
```

Comment:
are the HTTP method and WebSocket API method properties? What are they? How do you enter them? 
When there is one required and several optional properties, say something like this:

```suggestion
The `create` command has one required property, `name`, and two optional properties that allow you to specify an HTTP method and a WebSocket API method.
```
As I'm not sure if those are properties, we need to be clear about them.
Include instructions for how to enter the command string per previous comment.
If there are a few optional properties you can include a table. See the example in my PR for CLISnapshot.md or the one for CLISnapshotTask.md that is in branch PD-473.